lib/SPE.js

```diff
@@ -523,11 +522,11 @@ SPE.ShaderAttribute.prototype.setUpdateRange = function( min, max ) {
 SPE.ShaderAttribute.prototype.flagUpdate = function() {
 	'use strict';
 
-	var attr = this.bufferAttribute,
-		range = attr.updateRange;
+	var attr = this.bufferAttribute;
 
-	range.offset = this.updateMin;
-	range.count = Math.min( ( this.updateMax - this.updateMin ) + this.componentSize, this.typedArray.array.length );
+  const count = Math.min( ( this.updateMax - this.updateMin ) + this.componentSize, this.typedArray.array.length );
```

Comment:
It seems I commented on the commit last time, so it doesn't show up in the PR.
Can you use var here to be consistent in this file and use tab instead of spaces.

Reply:
done